src/logger.spec.ts

```diff
@@ -382,6 +382,31 @@ describe('Logger', () => {
     });
 });
 
+describe('Logger', () => {
+    const called = stub();
+    let infoFn: any;
+    beforeEach(() => {
+        infoFn = console.info;
```

Comment:
It is better to use the Sinon API: `const consoleInfo = stub(console, 'info');` and later on: `consoleInfo.restore();`.